ckanext/dia_theme/less/main.less

```diff
@@ -1,6 +1,6 @@
 // CKAN Base
-@import "../../../../ckan-core/ckan/public/base/less/variables.less";
-@import "../../../../ckan-core/ckan/public/base/less/bootstrap-variables.less";
+// @import "../../../../ckan-core/ckan/public-bs3/base/scss/_variables.scss";
+// @import "../../../../ckan-core/ckan/public-bs3/base/vendor/bootstrap/stylesheets/bootstrap/_variables.scss";
```

Comment:
Might need to look into this some more to make sure the less -> css processing still works